tests/integration/sugar/test_transaction.py

```diff
@@ -216,6 +216,37 @@ async def test_calculate_payment_fee(self, client):
         expected_fee = await get_fee(client)
         self.assertEqual(payment_autofilled.fee, expected_fee)
 
+    @test_async_and_sync(
+        globals(),
+        ["xrpl.transaction.autofill"],
+    )
+    # Autofill should populate the tx networkID and build_version from 1.11.0 or later.
+    # NetworkID field is populated only for networks where network_id > 1024
```

Comment:
These comments should be inside the tests, not above them

Reply:
fixed in 394d1fd